apps/server/src/modules/files-storage/controller/dto/file-storage.params.ts

```diff
@@ -102,3 +103,17 @@ export class CopyFilesOfParentPayload {
 	@ValidateNested()
 	target!: FileRecordParams;
 }
+
+export class PreviewParams {
+	@ApiProperty()
+	@IsInt()
+	width!: number;
+
+	@ApiProperty()
+	@IsInt()
+	height!: number;
```

Comment:
What does Image Magick do when you specify both width and height? Does it crop the image?

And is the height really needed? At least for the board you would always take the width of the surrounding element, wouldn't you?

Reply:
No by default it keeps the ratio and the image is not croped. We currently might not need the height but I would keep the endpoint a bit more generic.

Reply:
Is the image then distorted, when the specified width and height have a different ratio from the original image?

Reply:
No, it keeps the the aspect ratio dnd resizes the image to the smaller value of width and height.

Reply:
Put that in comment